tileview/src/main/java/com/qozix/tileview/tiles/TileCanvasViewGroup.java

```diff
@@ -242,14 +243,19 @@ void onRenderTaskCancelled() {
 
   void onRenderTaskPostExecute() {
     mIsRendering = false;
-    if( !mTransitionsEnabled ) {
-      cleanup();
-    }
-    if( mTileRenderListener != null ) {
-      mTileRenderListener.onRenderComplete();
-    }
-    invalidate();
-    requestRender();
+    mTileRenderHandler.post( new Runnable() {
```

Comment:
question: have you found a benefit to using throw-away anonymous instances of things like `Runnable` preferable to either A) creating a single instance as a member variable and passing that instance to `.post`, or B) implementing `Runnable` and passing bare instances e.g., `.post( new RenderTaskPostExecuteRunnable() );`

i'm not saying it has to be one way or another, or even different than what you've done - just asking your opinion.  i personally believe it to be most efficient (in this case) to use alternative A - define the `Runnable` once as a member variable of `TileCanvasViewGroup` and pass that to `Handler.post` each time, but am definitely open to other ways if there's a reason to.

@peterLaurence feel free to weigh in here too